packages/core/components/TooltipLink/TooltipLink.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Tooltip as ReactTooltip } from 'react-tooltip'
+import './TooltipLink.css'
+
+const TooltipLink = ({ linkText, href = null, tooltipOpacity = 100 }) => {
+  return (
+    <>
+      <a
+        className='tooltip-link'
+        data-tooltip-content='Data is
+               suppressed to maintain statistical reliability.
+                This occurs when the number of respondents or 
+                reported values does not meet the minimum
+                 reporting threshold.'
+        data-tooltip-id='supression-tooltip'
+        href={href}
+      >
+        {linkText}
+      </a>
+
+      <ReactTooltip
+        id='supression-tooltip'
```

Comment:
I don't think you need these individual `<ReactTooltip />` components for each tooltip. You can likely reference the component thats found at the bottom of the Chart file. Right now I think thats ln 1538 in LinearChart:

```.js
          <ReactTooltip
            id={`cdc-open-viz-tooltip-${runtime.uniqueId}`}
            ...
```